### PR TITLE
[Flutter-Parent][MBL-14080] Report problem dialog for small devices

### PR DESCRIPTION
**Pull request by CalvinKern**
TIL: In flutter, you can wrap a dialog in a scroll view

To test: See ticket for how small screens don't view the report problem dialog well.
Should also test on a big screen size to make sure it still looks good for the majority of devices.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1025203&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/utils/common_widgets/error_report/error_report_dialog.dart [hunk 1]
```diff
@@ -158,26 +171,27 @@ class _ErrorReportDialogState extends State<ErrorReportDialog> {
...
-                    // NO!  This messes up dpav-nav
-                    //_focusScopeNode.requestFocus(FocusNode());
-                  },
-                  value: severityOptions.firstWhere((option) => option.severity == _selectedSeverity),
-                  items: severityOptions.map((option) {
-                    return DropdownMenuItem<_SeverityOption>(value: option, child: Text(option.label));
-                  }).toList(),
+              if (!widget.hideSeverityPicker) Text(L10n(context).reportProblemSeverity),
+              if (!widget.hideSeverityPicker)
+                Container(
```
Comment (joehoag) [inline apps/flutter_parent/lib/utils/common_widgets/error_report/error_report_dialog.dart]: Sometimes auto-formatting leaves me scratching my head...

Comment (CalvinKern) [reply apps/flutter_parent/lib/utils/common_widgets/error_report/error_report_dialog.dart]: Yeah, not sure why this changed on this pass... Hopefully things become more stable as we keep on it.

**Pull request merged**